index.html

```diff
@@ -1879,31 +1879,212 @@ <h3>Simple System Lifecycle</h3>
+                        installing and uninstalling applications, reconfiguring
+                        settings, etc.
+                        Note that a device can be operational in its own native
+                        protocol suite and managed by a WoT gateway, or can be
+                        operational for WoT (which may be an application on the device),
+                        or may be operational for Wot and directly provisioned for
+                        WoT.
+                    </li>
+                    <li>
+                        <strong>Maintenance</strong>: the device operational state
+                        is interrupted for updating its software and/or configuration.
+                    </li>
+                    <li>
+                        <strong>Destroyed</strong>: the device has been wiped out
+                        of all data and software. Hardware kill features may
+                        be activated. The device may be physically destroyed and
+                        never used again. This state is relevant for device management purposes.
+                        It does not exist in OneM2M, LwM2M, OCF, T2TRG and is
+                        called <em>End-of-life</em> in OPC-UA and Anima.
+                    </li>
+                </ul>
+              </p>
+              <figure id="lifecycle-1">
+                <img src="images/architecture/device-lifecycle-1.svg"
+                        srcset="images/architecture/device-lifecycle-1.svg"
+                        class="wot-arch-diagram" alt="Device Lifecycle" />
+                <figcaption>Device Lifecycle</figcaption>
+              </figure>
+              <p>
+                The most typical transitions between lifecycle states are the
+                following:
+                <ul>
+                    <li>
+                        <strong>Bootstrapping</strong> (or onboarding): the device
+                        is provisioned with an identity, the chain of trust is
+                        established, e.g. secure storage, keys, certificates.
+                        This may involve various solutions, such as manufacturer
+                        certificates, out-of-band key provisioning, connecting
+                        to a provisioning server (a quite common scenario). When
+                        provisioning directly to WoT, the device may be registered
+                        with a <a>Thing Directory</a> in this stage.
+                    </li>
+                    <li>
+                        <strong>Provisioning, configuration, commissioning</strong>:
+                        the device is provisioned with all resources needed for
+                        its operation (services, applications, access control,
+                        databases etc), and these resources are configured for
+                        operation. Also, the device may be commissioned in a
+                        given environment.
```

Comment:
Also, the device may be commissioned in a given environment, and this may include (for example) registration with discovery services.